gradle-plugin/src/main/groovy/io/realm/gradle/Realm.groovy

```diff
@@ -48,6 +49,9 @@ class Realm implements Plugin<Project> {
         }
 
         project.android.registerTransform(new RealmTransformer())
+        if (!isAndroidLib) {
```

Comment:
This just to remove the optional API as the "almost last" step before packing apk. In an android lib project this won't be needed as it will be used in a app project and that may have a required dependency.